recipe/gen_patch_json.py

```diff
@@ -1970,6 +1970,14 @@ def _gen_new_index_per_key(repodata, subdir, index_key):
                 ):
             _replace_pin("python >=3.7", "python >=3.8", record["depends"], record)
 
+        # add  as run_constrained for cling
+        if (
+            record_name == "cling"
+            and record['version'] >= "0.8"
+        ):
+            record.setdefault('constrains', []).extend((
+                "gxx_linux-64 !=9.5.0  # [clangdev == \"5.*\"] - sysroot_linux-64  # [linux]",
```

Comment:
```suggestion
                "gxx_linux-64 !=9.5.0",
```

I don't think that selectors work here. Can you update your patching?